metrics-operator/go.mod

```diff
@@ -43,6 +45,8 @@ require (
 	github.com/coreos/go-semver v0.3.1 // indirect
 	github.com/coreos/go-systemd/v22 v22.5.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/elastic/go-elasticsearch/v7 v7.17.10
```

Comment:
i think the v7 dependency is unused and could be removed

Reply:
sure